src/components/ProductGridHero/ProductGridHero.js

```diff
@@ -37,7 +35,7 @@ export default class ProductGridHero extends Component {
       <section className={classes.heroGridContainer}>
         <Grid container spacing={24}>
           <Grid item xs={12}>
-            <Img isHero src={heroMediaUrl} />
+            <ProgressiveImage src={heroMediaUrl} className={classes.heroContainer} />
```

Comment:
Why doesn't a wrapper work here?

```jsx
<div className={classes.heroContainer}>
  <ProgressiveImage src={heroMediaUrl} />
</div>
```

Or even just put the className on `<Grid item`?

I'd prefer to avoid overriding component library styles using className, especially because that mixes styled-components with MUI, but also because people are following these patterns and it's not a pattern we want to encourage.

Reply:
I tried for quite a while to get something like a wrapper element working (including exactly what you have above). It's because the element that is given the padding by the component library that needs to be overwritten is the `ProgressiveImage` itself, and even with negative padding the desired look is seemingly unachievable without overwriting that property on that element.